website/src/data/users.tsx

```diff
@@ -776,6 +776,14 @@ const Users: User[] = [
     website: 'https://khyron-realm.com',
     source: 'https://github.com/khyron-realm/khyron-realm-website',
     tags: ['opensource'],
+  },
+   {
+    title: 'Kaustubh Kulkarni',
+    description: "Portfolio Website",
+    preview: require('./showcase/woodpecker.png'),
+    website: 'https://www.kaustubhk24.com/',
+    source: https://www.kaustubhk24.com/',
+    tags: ['personal','portfolio],
```

Comment:
1. Multiple syntax errors. Please fix.
2. Please upload your site's screenshot and update the preview path here.
3. Your site's GitHub link is pointing to ours. Please change that to your repo or remove it.
4. You have to change the source link here to either `null` if you are not opensource, or your website's repo.
5. We don't have a "portfolio" tag.